### PR TITLE
Updated index with functions to call survey gizmo api

### PR DESCRIPTION
**Pull request by RyanPnayR**

### CODE REVIEW

**Comment by aloisbarreras** [comment]
@RyanPnayR you indentation still looks like 8 spaces instead of 4. Also I see a bunch of empty newlines. Get all the formatting things cleaned up and let me know when it's ready.

File: README.md [hunk 1]
```diff
@@ -1,89 +1,125 @@
...
 ##Method ID's
-These IDs should be used with the methods they are required for. 
+These IDs should be used with the methods they are required for.
 
 ###Survey ID
 ```javascript
-surveyId: "123456", 
+surveyId: "123456",
 ```
 ###Account Temas ID
```
Comment (trong-le) [inline README.md]: spelling

**Comment by jlaneve** [comment]
@trong-le fixed

**Comment by trong-le** [comment]
@jlaneve Can you add descriptions also about the method ID's, pretty much just where to find them

**Comment by jlaneve** [comment]
@trong-le not 100% sure of this (I didn't build the integration), but I think you'd just find the respective id with the method that lists the items.

**Pull request merged**